src/app/search/loadouts/search-filters/freeform.ts

```diff
@@ -22,6 +44,31 @@ const freeformFilters: FilterDefinition<
       return (loadout) => test(loadout.name);
     },
   },
+  {
+    keywords: ['subclass'],
+    description: tl('LoadoutFilter.Name'),
+    format: ['simple', 'freeform'],
```

Comment:
I think you can remove 'simple' - that'd be for things like `is:subclass` which would be a different search.

Reply:
Done